libs/openFrameworks/gl/ofGLProgrammableRenderer.cpp

```diff
@@ -1315,12 +1321,46 @@ void ofGLProgrammableRenderer::bind(const ofFbo & fbo, bool setupPerspective){
 }
 
 //----------------------------------------------------------
-void ofGLProgrammableRenderer::unbind(const ofFbo & fbo){
-	glBindFramebuffer(GL_FRAMEBUFFER, 0);
+void ofGLProgrammableRenderer::end(const ofFbo & fbo){
+	fbo.unbind();
```

Comment:
Hey @arturoc, can you explain a little more what the thread-issue is? 

I've got a bit of time to fix things if neccessary, but it's a bit of a pain keeping this mergeable. And since MRT is broken since (or shortly before) multiwindow it would be nice to have it working again for 0.9.

I see that multithreading was discussed in #3260, but i'm not sure how current the discussion there is.